FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Basic_TLS/demo_config.h

```diff
@@ -76,6 +76,9 @@
  * Settings/Custom Endpoint, or using the describe-endpoint REST API (with
```

Comment:
Can AWS IoT endpoint work for a basic TLS demo? If not, I think it is better to consider mqtt_broker_setup as the first option

Reply:
Good point. I think it's a copy-paste error.